tests/unittests/utils/test_mapanything.py

```
@@ -0,0 +1,1018 @@
+        _patch_infer(model, _mock_pred())
+        with _patch_load_images():
+            model._predict_all([Image.new("RGB", (64, 64))])
+        call_kwargs = model._model.infer.call_args[1]
+        assert call_kwargs["apply_mask"] is True
+        assert call_kwargs["mask_edges"] is True
+
+    def test_temp_file_cleaned_up(self):
+        model = _make_model("depth")
+        _patch_infer(model, _mock_pred())
+        created_files = []
+        original_load = None
```

Ruff (0.14.14)

[error] Local variable `original_load` is assigned to but never used

Remove assignment to unused variable `original_load`

(F841)